CMakeLists.txt

```diff
@@ -18,6 +18,17 @@ FetchContent_Declare(googletest
 FetchContent_MakeAvailable(googletest)
 
 
```

Comment:
There must be some instructions for how to enabling SWIG etc.
As default must the SWIG option be disabled.

Reply:
Will update README. Default for SWIG is off